lib/WorseReflection/Tests/Integration/Core/Inference/NodeContextResolverTest.php

```diff
@@ -1175,7 +1175,7 @@ private function resolveNodeAtOffset(
         $nameResolver = new NodeToTypeConverter($reflector, $this->logger());
         $resolver = new NodeContextResolver(
             $reflector,
-            new DocblockParserFactory($reflector),
+            new PHPStanDocblockParserFactory($reflector),
```

Comment:
Parser used in tests is now hardcoded. I have no good idea, those parses could be iterated to execute test suite with both of them

Reply:
This is the weaker set of tests, the `Inference/SelfTest.php` is much easier to manage and if it passes the chances are we're good. So this could remain with the original parser for now.